src/selectors/disabled.js

```diff
@@ -163,9 +166,21 @@ export const getNodeDisabled = createSelector(
     visibleModularPipelineInputsOutputs,
     disabledModularPipeline,
     slicedPipeline,
-    isSliceApplied
+    isSliceApplied,
+    isModularPipelineFocusMode,
+    modularPipelinesTree
   ) =>
     arrayToObject(nodeIDs, (id) => {
+      const rootChildren = modularPipelinesTree?.['__root__']?.children || [];
```

Comment:
I was thinking maybe when we add modularPipelines on expand, we just don't add '__root__' pipeline in flowchart-primary.js.